packages/core/src/routes-me/user.ts

```diff
@@ -129,16 +130,27 @@ export default function userRoutes<T extends AuthedMeRouter>(
 
   router.post(
     '/password',
-    koaGuard({ body: object({ password: string().min(1) }) }),
+    koaGuard({ body: object({ password: string().min(1) }), status: [204, 400] }),
     async (ctx, next) => {
       const { id: userId } = ctx.auth;
       const { password } = ctx.guard.body;
 
-      const { isSuspended } = await findUserById(userId);
+      const user = await findUserById(userId);
 
-      assertThat(!isSuspended, new RequestError({ code: 'user.suspended', status: 401 }));
+      assertThat(!user.isSuspended, new RequestError({ code: 'user.suspended', status: 401 }));
 
-      await checkVerificationStatus(userId, null);
+      const [signInExperience] = await Promise.all([
+        findDefaultSignInExperience(),
+        checkVerificationStatus(userId, null),
+      ]);
+      const passwordPolicyChecker = new PasswordPolicyChecker(signInExperience.passwordPolicy);
```

Comment:
I think we should also pass user to the policy checker

Reply:
how?

Reply:
this constructor has no user parameter, and the user is passed right after this line. so how can i pass user here and why should i do it?

Reply:
Oh yes, the user is not passed through the constructor, but the `.check()` function. My bad.